package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "google-closure-library",
+  "description": "Google's common JavaScript library",
+  "version": "1.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google/closure-library.git"
+  },
+  "author": "The Closure Library Authors",
+  "license": "Apache-2.0",
```

Comment:
you should probably set 'main' to do something reasonable? like maybe load the nodejs bootstrapper and return 'goog'?